openfl/protocols/director.proto

```diff
@@ -133,6 +191,10 @@ service FederationDirector {
     rpc GetExperimentData (GetExperimentDataRequest) returns (stream ExperimentData) {}
     rpc EnvoyHealthCheck (EnvoyStatus) returns (EnvoyHealthCheckResponse) {}
 
+    // Experiments RPCs
```

Comment:
These RPCs are also made by front-end clients. In this sense, those are also API RPCs (better rename to FRONTEND RPCs)

Reply:
Separate issue was created https://github.com/intel/openfl/issues/222